ex02.exs

```diff
@@ -94,11 +97,13 @@ defmodule Ex02 do
     # « your code goes here»
     # You can give an agent a name by adding the
     # `name: @global_name` option to start_link
+    Agent.start_link(fn-> initial_value end, name: @global_name)
   end
 
   def global_next_value do
     # « your code goes here»
     # and you can use that name instead of a pid
     # in calls such as `get_and_update`
+    Agent.get_and_update(:my_global_agent, fn value -> {value, value+1} end)
```

Comment:
So close! Why not use `@global_name` here, too. The idea is that if you use it in both places, then anyone changing the code only has to change one thing.

# -1